tests/test_commands.py

```diff
@@ -0,0 +1,72 @@
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import CommandError, call_command
+from django.test import TestCase
+
+import pytest
+
+from digid_eherkenning.models import DigidConfiguration, EherkenningConfiguration
+
+from .conftest import DIGID_TEST_METADATA_FILE_SLO_POST, EHERKENNING_TEST_METADATA_FILE
+
+
+@pytest.mark.usefixtures("digid_config_defaults", "temp_private_root")
+class UpdateStoredMetadataTests(TestCase):
```

Comment:
can you please write future tests in libraries as pytest instead of unittest.TestCase?

imo this should've been simply at the module level:

```py
def test_command_triggers_xml_fetching_when_digid(patcher):
    ...
```